src/app/app.component.css

```diff
@@ -59,6 +68,31 @@ img.product_image {
   color: black;
 }
 
+.header{
+  width: 100%;
+  height: auto;
+  background-color: ;
```

Comment:
Maybe delete this line if it’s not in use

Reply:
Cleaned it up. 👍